pages/Documents/Reporting/OutboundReportingAPI/overview.md

```diff
@@ -405,388 +409,397 @@ Click [**Transaction**](https://proactive-messaging.z1.fs.liveperson.com/api/api
- "message": "<<error Message>>"
+  "code": 0,
+  "requestTraceId": "5102ce1f-35bd-4e5e-9bd3-bacdfd28dd3a",
+  "message": "<<error Message>>"
 }
 ```
 
 ### Frequently Asked Questions
 
-<strong>1. What authorizations are supported by reporting API?</strong>
+#### 1. What authorizations are supported by reporting API?
 
-Following authentication and authorization are supported by reporting API
+The following authentication and authorization methods are supported by the reporting API:
 
-* OAuth 2.0 (Recommended)
+* OAuth 2.0 (recommended)
 * OAuth 1.0
 * LE Bearer Token
 
 - For OAuth 2.0, refer to [this document](https://developers.liveperson.com/connector-api-send-api-authorization-and-authentication.html#get-appjwt)
-- For OAuth 1.0, refer to [this document Step 8](https://developers.liveperson.com/retrieve-api-keys-create-a-new-api-key.html), Keys will be provided separately
+- For OAuth 1.0, refer to [step 8 in the OAuth 1.0 API keys documentation](oauth-1-0-api-keys.html); keys will be provided separately
 
-<strong>2. Who can access the Outbound Reporting API?</strong>
+#### 2. Who can access the Outbound Reporting API?
 
-All Brands who use Proactive Messaging version 2.0 and Connect To Messaging version 2.0 have access to the Outbound Reporting API service.
+All brands who use Proactive Messaging version 2.0 and Connect To Messaging version 2.0 have access to the Outbound Reporting API service.
 
-<strong>3. What is the rate limit for the API’s?</strong>
+#### 3. What is the rate limit for the APIs?
 
 - Transaction API, rate: 10 TPS
 - Analytics API, rate: 10 TPS
 - Proactive Campaign API, rate: 10 TPS
 - Proactive Campaign Analytics API, rate: 10 TPS
 
-<strong>4. How can a brand find out which version of the Proactive Messaging or Connect To Messaging it uses ?</strong>
+#### 4. How can a brand find out which version of the Proactive Messaging or Connect To Messaging it uses ?
 
 For Proactive Messaging:
-- Sign in to [this url](https://proactive-messaging.fs.liveperson.com) or click on the quick launch icon from Conversation Cloud for Proactive Messaging.
+
+- Sign in to [Proactive Messaging](https://proactive-messaging.fs.liveperson.com) or click on the quick launch icon from Conversation Cloud for Proactive Messaging.
 - Click on the user icon at top right corner and see the version.
 
 For Connect To Messaging:
-- Sign in to [this url](https://connect-to-messaging.fs.liveperson.com) or click on the quick launch icon from Conversation Cloud for Connect To Messaging.
+
+- Sign in to [Connect To Messaging](https://connect-to-messaging.fs.liveperson.com) or click on the quick launch icon from Conversation Cloud for Connect To Messaging.
 - Click on the user icon at top right corner and see the version.
 
-<strong>6. What is a LP data retention policy? And how long data is persisted for Outbound Reporting API?</strong>
+#### 6. What is a LP data retention policy? And how long data is persisted for Outbound Reporting API?
```

Comment:
What is a LP data retention policy? >>> either change to "an" or "the"

Reply:
Changed it to “the LivePerson data retention policy”. Not a fan of the “LP” abbreviation in public docs—have replaced this in the past, might be good to take note on to refactor across the DC.